sale_rental/views/sale_rental.xml

```diff
@@ -0,0 +1,193 @@
+                    decoration-info="state == 'ordered'"
+                    decoration-warning="state == 'out'"
+                    decoration-muted="state == 'cancel'"
+                    decoration-danger="state == 'sold'"
+                />
+        </tree>
+    </field>
+</record>
+
+<record id="sale_rental_search" model="ir.ui.view">
+    <field name="name">sale.rental.search</field>
+    <field name="model">sale.rental</field>
+    <field name="arch" type="xml">
+        <search string="Search Rentals">
+            <field name="start_order_id" />
+            <field name="partner_id" />
+            <field name="rented_product_id" />
+            <filter
+                    name="ordered"
+                    string="Ordered"
+                    domain="[('state', '=', 'ordered')]"
+                />
+            <filter name="out" string="Out" domain="[('state', '=', 'out')]" />
+            <filter
+                    name="sell_progress"
+                    string="Sell in progress"
+                    domain="[('state', '=', 'sell_progress')]"
+                />
+            <filter name="sold" string="Sold" domain="[('state', '=', 'sold')]" />
+            <filter name="in" string="Back In" domain="[('state', '=', 'in')]" />
+            <group string="Group By" name="groupby">
+                <filter
+                        name="partner_groupby"
+                        string="Customer"
+                        context="{'group_by': 'partner_id'}"
+                    />
+                <filter
+                        name="state_groupby"
+                        string="State"
+                        context="{'group_by': 'state'}"
+                    />
+                <filter
+                        name="start_date_groupby"
+                        string="Start Date"
+                        context="{'group_by': 'start_date'}"
+                    />
+                <filter
+                        name="end_date_groupby"
+                        string="End Date"
+                        context="{'group_by': 'end_date'}"
+                    />
+            </group>
+        </search>
+    </field>
+</record>
+
+<record id="sale_rental_pivot" model="ir.ui.view">
+    <field name="name">sale.rental.pivot</field>
+    <field name="model">sale.rental</field>
+    <field name="arch" type="xml">
+        <pivot>
+            <field name="rented_product_id" type="row" />
+            <field name="start_date" interval="month" type="col" />
+            <field name="rental_qty" type="measure" />
+        </pivot>
+    </field>
+</record>
+
+
+
+
+<record id="sale_rental_action" model="ir.actions.act_window">
+    <field name="name">Rentals</field>
+    <field name="res_model">sale.rental</field>
+    <field name="view_mode">tree,form,pivot</field>
+</record>
+
+<menuitem
+        id="sale_rental_menu"
+        parent="sale.sale_order_menu"
+        action="sale_rental_action"
+        sequence="2"
```

Comment:
```suggestion
        sequence="25"
```
The menu is shown as first on sale which does not seem relevant, better to have it later in the menu list.